lib/algoliasearch-rails.rb

```diff
@@ -821,7 +806,7 @@ def algolia_ensure_init(options = nil, settings = nil, index_settings = nil)
                    index_settings.delete(:slaves) ||
                    index_settings.delete('slaves')
         index_settings[used_slaves ? :slaves : :replicas] = replicas unless replicas.nil? || options[:inherit]
-        @algolia_indexes[settings].set_settings(index_settings)
+        @algolia_indexes[settings].set_settings!(index_settings)
```

Comment:
Why do we change it to `set_settings!` here instead of using the method without `!`?

Reply:
I don't understand why we wouldn't wait for the settings to be updated in the first place, this is why

Reply:
Seeing how it's used I think we can indeed wait for the settings to pass 👍